ocaml/idl/markdown_backend.ml

```diff
@@ -43,8 +43,6 @@ let compare_case_ins x y =
   compare (String.lowercase_ascii x) (String.lowercase_ascii y)
 
 let escape s =
-  let open Xapi_stdext_std.Xstringext in
```

Comment:
There is a project called `cmarkit` that can be use to generate Markdown from OCaml, perhaps in the future we might want to use it instead of generating markdown by hand here.
It can both parse and render Markdown https://ocaml.org/p/cmarkit/latest/doc/index.html